### PR TITLE
Adding localization for Portuguese language

### LINKED ISSUE #857: Add locale pt-br. I can help with translation.
Thank you for reporting an issue, suggesting an enhancement, or asking a question. We appreciate your feedback - to help the team understand your
needs please complete the below template to ensure we have the details to help. Thanks!

**Please check out the [documentation](https://pnp.github.io/sp-dev-fx-controls-react/) to see if your question is already addressed there. This will help us ensure our documentation is up to date.**

#### Category
[X ] Enhancement

[ ] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [  2.0.0   ]

_If you are not using the latest release, please update and see if the issue is resolved before submitting an issue._

#### Expected / Desired Behavior / Question
_If you are reporting an issue please describe the expected behavior. If you are suggesting an enhancement please
describe thoroughly the enhancement, how it can be achieved, and expected benefit. If you are asking a question, ask away!_

#### Observed Behavior
_If you are reporting an issue please describe the behavior you expected to occur when performing the action. If you are making a
suggestion or asking a question delete this section._

#### Steps to Reproduce
_If you are reporting an issue please describe the steps to reproduce the bug in sufficient detail to allow testing. If you are making
a suggestion or asking a question delete this section._

#### Submission Guidelines
_Delete this section after reading_
- All suggestions, questions and issues are welcome, please let us know what's on your mind.
- Remember to include sufficient details and context.
- If you have multiple suggestions, questions, or bugs please submit them in separate issues so we can track resolution.

Thanks!

### PR DESCRIPTION
**Pull request by kunj-sangani**
Adding localization for Portuguese language

| Q               | A
| --------------- | ---
| Bug fix?        | [x]
| Related issues?  | fixes #857 

#### What's in this Pull Request?

Adding localization for Portuguese language which was mentioned in #857

### CODE REVIEW

File: src/loc/pt-br.ts [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    DatePickerDayLongTuesday: "Terça-feira",
+    DatePickerDayShortTuesday: "Ter",
+    DatePickerDayLongWednesday: "Quarta-feira",
+    DatePickerDayShortWednesday: "Casar",
+    DatePickerDayLongThursday: "Quinta-feira",
+    DatePickerDayShortThursday: "Qui",
+    DatePickerDayLongFriday: "Sexta-feira",
+    DatePickerDayShortFriday: "Sex",
+    DatePickerDayLongSaturday: "Sábado",
+    DatePickerDayShortSaturday: "Sentado",
```
Comment (aaclage) [inline src/loc/pt-br.ts:206]: @joelfmrodrigues 
I don't think this is correct.  DatePickerDayShortSaturday: "Sentado" Sentado -> means seated... should be "Sáb"

File: src/loc/pt-br.ts [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+    DatePickerMonthLongDecember: "Dezembro",
+    DatePickerMonthShortDecember: "Dezembro",
+    DatePickerDayLongSunday: "Domigo",
+    DatePickerDayShortSunday: "Sol",
+    DatePickerDayLongMonday: "Segunda-feira",
+    DatePickerDayShortMonday: "Seg",
+    DatePickerDayLongTuesday: "Terça-feira",
+    DatePickerDayShortTuesday: "Ter",
+    DatePickerDayLongWednesday: "Quarta-feira",
+    DatePickerDayShortWednesday: "Casar",
```
Comment (aaclage) [inline src/loc/pt-br.ts:200]: @joelfmrodrigues
I don't think this is correct. DatePickerDayShortSaturday: "Casar" -> means married... should be "Qua" i think...

File: src/loc/pt-br.ts [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+    DatePickerMonthLongSeptember: "Setembro",
+    DatePickerMonthShortSeptember: "Setembro",
+    DatePickerMonthLongOctober: "Outubro",
+    DatePickerMonthShortOctober: "Outubro",
+    DatePickerMonthLongNovember: "Novembro",
+    DatePickerMonthShortNovember: "Novembro",
+    DatePickerMonthLongDecember: "Dezembro",
+    DatePickerMonthShortDecember: "Dezembro",
+    DatePickerDayLongSunday: "Domigo",
+    DatePickerDayShortSunday: "Sol",
```
Comment (aaclage) [inline src/loc/pt-br.ts:194]: @joelfmrodrigues
I don't think this is correct. DatePickerDayShortSaturday: "Sol" -> means sun... should be "Dom" i think...

File: src/loc/pt-br.ts [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+    StandardColorGreen: "Verde",
+    StandardColorLightblue: "Azul claro",
+    StandardColorBlue: "Azul",
+    StandardColorDarkblue: "Azul escuro",
+    StandardColorPurple: "Roxo",
+
+    DatePickerMonthLongJanuary: "Janeiro",
+    DatePickerMonthShortJanuary: "Jan",
+    DatePickerMonthLongFebruary: "Fevereiro",
+    DatePickerMonthShortFebruary: "Fevereiro",
```
Comment (aaclage) [inline src/loc/pt-br.ts:172]: @joelfmrodrigues DatePickerMonthShort should be short "Fev" but appear long, the same until December.

**Comment by joelfmrodrigues** [comment]
@kunj-sangani Many thanks for this! The comments that Andre made are valid. Could you please update the PR?

**Comment by joelfmrodrigues** [comment]
@aaclage many thanks for the comments!

**Comment by kunj-sangani** [comment]
@joelfmrodrigues  Sure let me update as mentioned by @aaclage  and then raise the PR again hope that helps :)

File: src/loc/pt-br.ts [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+      "L_RelativeDateTime_XHoursIntervals": "1 || 2-",
+      "L_RelativeDateTime_DayAndTime": "{0} em {1}",
+      "L_RelativeDateTime_XDaysFuture": "{0} dias a partir de agora || {0} dias a partir de agora",
+      "L_RelativeDateTime_XDays": "{0} dia atrás || {0} dias atrás",
+      "L_RelativeDateTime_XDaysFutureIntervals": "1 || 2-",
+      "L_RelativeDateTime_XDaysIntervals": "1 || 2-",
+      "L_RelativeDateTime_Today": "Hoje"
+    },
+    "SendEmailTo": "Envie um e-mail para {0}",
+    "StartChatWith": "Inicie um bate-papo com {0}",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:36]: "Inicie um bate-papo com {0}" should be "Inicie uma conversa com {0}"

File: src/loc/pt-br.ts [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+    "TaxonomyPickerInLabel": "no",
+    "TaxonomyPickerTermSetLabel": "Conjunto de termos",
+    "TaxonomyPickerTermsNotFound": "Os próximos termos selecionados não foram encontrados no repositório de termos: {0}",
+    "TaxonomyPickerInvalidTerms": "Corrija os termos inválidos: {0}",
+
+    peoplePickerComponentTooltipMessage: "Selecionador de Pessoas",
+    peoplePickerComponentErrorMessage: "Campo obrigatório",
+    peoplePickerSuggestionsHeaderText: 'Pessoas sugeridas',
+    peoplePickerLoadingText: 'Carregando',
+    PeoplePickerSearchText: 'Buscando usuários',
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:52]: "Buscando usuários" should be "Procurando utilizadores"

File: src/loc/pt-br.ts [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+
+    mapsErrorMessage: 'Ocorreu um erro ao carregar o mapa',
+    mapsLoadingText: 'Carregando',
+    mapsSearchButtonText: 'Procurar',
+    mapsTitlePrefix: 'Mapa de',
+
+    ListViewFilterLabel: "Filtre a lista",
+
+    HeaderNormalText: "Texto normal",
+    HeaderH2: "Cabeçallho 1",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:85]: "Cabeçallho" should be "Cabeçalho"

File: src/loc/pt-br.ts [hunk 8]
```diff
@@ -0,0 +1,358 @@
...
+    StandardColorLightblue: "Azul claro",
+    StandardColorBlue: "Azul",
+    StandardColorDarkblue: "Azul escuro",
+    StandardColorPurple: "Roxo",
+
+    DatePickerMonthLongJanuary: "Janeiro",
+    DatePickerMonthShortJanuary: "Jan",
+    DatePickerMonthLongFebruary: "Fevereiro",
+    DatePickerMonthShortFebruary: "Fevereiro",
+    DatePickerMonthLongMarch: "Marcha",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:173]: Março

File: src/loc/pt-br.ts [hunk 9]
```diff
@@ -0,0 +1,358 @@
...
+    DatePickerMonthShortMarch: "Março",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:174]: Mar

File: src/loc/pt-br.ts [hunk 10]
```diff
@@ -0,0 +1,358 @@
...
+    DatePickerMonthLongApril: "Abril",
+    DatePickerMonthShortApril: "Abril",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:176]: Abr

File: src/loc/pt-br.ts [hunk 11]
```diff
@@ -0,0 +1,358 @@
...
+    FolderBackPlate: "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/foldericons/folder-large_backplate.svg",
+    FolderFrontPlate: "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/foldericons/folder-large_frontplate_nopreview.svg",
+    FolderIconUrl: "https://spoprod-a.akamaihd.net/files/odsp-next-prod_2019-01-11_20190116.001/odsp-media/images/itemtypes/20/folder.svg",
+    FolderLabelTemplate: "{0}, Pasta, Modificado {1}, editado por {2}, {3} itens, Privado",
+    FromLinkLinkLabel: "De um link",
+    ImageAltText: ". {0} Imagem",
+    ImageAriaLabelTemplate: ". {0} Imagem",
+    ImageLayoutPlaceholderText: "Layout",
+    ImageSizePlaceholderText: "Tamanho da imagem",
+    ItemChildCountField: "Contagem de itens filhos",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:247]: "itens" should be "items"

File: src/loc/pt-br.ts [hunk 12]
```diff
@@ -0,0 +1,358 @@
...
+    LayoutOptionAll: "Tudo",
+    LayoutOptionSquare: "Praça",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:249]: "Praça" should be "Quadrado"

File: src/loc/pt-br.ts [hunk 13]
```diff
@@ -0,0 +1,358 @@
...
+    NoExternalLinksValidationMessage: "Só suportamos links para arquivos em sua própria organização.",
+    NoImageValidationMessage: "Este não é um link para um tipo de arquivo que suportamos. Você só pode criar um link para uma imagem.",
+    NoRecentFiles: "Nenhum arquivo recente",
+    NoRecentFilesDescription: "Tente selecionar um arquivo do seu site ou carregue um do seu dispositivo.",
+    NoResultsBadEnglish: "*** Nenhum resultado encontrado. Tente alterar as opções de filtro",
+    ODModifiedField: "Modificado",
+    ODPhotoIconUrl: "https://spoprod-a.akamaihd.net/files/odsp-next-prod_2019-01-18_20190124.001/odsp-media/images/itemtypes/16_2x/photo.png",
+    ODRowArialLabelTemplate: "{0}, {1}, Modificado {2}, editado por {3}, {4}, {5}",
+    OneDriveConfirmDialogBody: "Este item é do seu site OneDrive. Arquivos e pastas no OneDrive são privados, a menos que você os compartilhe. Você compartilhou este arquivo com membros do seu site para que eles possam acessá-lo?",
+    OneDriveConfirmDialogTitle: "Apenas checando...",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:283]: "Apenas checando..." should be """Apenas validando..."

File: src/loc/pt-br.ts [hunk 14]
```diff
@@ -0,0 +1,358 @@
...
+    OpenButtonLabel: "Abrir",
+    PhotoIconUrl: "https://spoprod-a.akamaihd.net/files/odsp-next-prod_2019-01-11_20190116.001/odsp-media/images/itemtypes/20_2x/photo.png",
+    OrgAssetsTabLabel: "Imagens e arquivos fornecidos por sua organização",
+    PoweredByBing: "Powered by Bing",
+    RecentDocumentsHeader: "Documentos recentes",
+    RecentImagesHeader: "Imagens recentes",
+    RecentLinkLabel: "Recente",
+    SearchBoxPlaceholder: "Pesquisa na internet",
+    SearchResultAlt: "Resultado de imagem para {0}.",
+    SearchResultAriaLabel: "Pressione Enter para abrir a fonte da imagem em uma nova guia.",
```
Comment (joelfmrodrigues) [inline src/loc/pt-br.ts:298]: "em uma nova guia" should be "em um novo separador"

**Comment by joelfmrodrigues** [comment]
Sorry if you received spam from my comments. Was reviewing it as if it was pt-pt and only at the end noticed it was pt-br...my bad

**Comment by joelfmrodrigues** [comment]
@kunj-sangani Also, can you please update target branch to be dev-v2?

**Comment by kunj-sangani** [comment]
Hi @joelfmrodrigues ,

Sure I will create a Pull request there

**Comment by joelfmrodrigues** [comment]
@kunj-sangani I just noticed that you pushed many other changes to the PR, possibly by accident. Could you please review and separate from this PR when you get the chance?

**Comment by kunj-sangani** [comment]
@joelfmrodrigues Sorry Let me split the Pull request

**Comment by joelfmrodrigues** [comment]
@kunj-sangani is this PR still valid?

**Comment by kunj-sangani** [comment]
@joelfmrodrigues 
No it can be discarded

**Pull request closed without merging**